tests/aws/services/lambda_/test_lambda_common.py

```diff
@@ -132,12 +132,16 @@ def _invoke_with_payload(payload):
             "$..environment.AWS_EXECUTION_ENV",  # Only rust runtime
             "$..environment.LD_LIBRARY_PATH",  # Only rust runtime (additional /var/lang/bin)
             "$..environment.PATH",  # Only rust runtime (additional /var/lang/bin)
-            "$..CodeSha256",  # works locally but unfortunately still produces a different hash in CI
             "$..environment.LC_CTYPE",  # Only python3.11 (part of a broken image rollout, likely rolled back)
             # Newer Nodejs images explicitly disable a temporary performance workaround for Nodejs 20 on certain hosts:
             # https://nodejs.org/api/cli.html#uv_use_io_uringvalue
             # https://techfindings.net/archives/6469
             "$..environment.UV_USE_IO_URING",  # Only Nodejs runtimes
+            # Only Dotnet8
+            "$..environment.DOTNET_CLI_TELEMETRY_OPTOUT",
+            "$..environment.DOTNET_NOLOGO",
+            "$..environment.DOTNET_RUNNING_IN_CONTAINER",
+            "$..environment.DOTNET_VERSION",
```

Comment:
So those are only set in LocalStack, but not in AWS?

Reply:
correct 👍 
(I can sneak in a clarifying comment in a later PR)